packages/flutter_tools/lib/src/doctor.dart

```diff
@@ -434,12 +434,26 @@ class Doctor {
         DoctorResultEvent(validator: validator, result: result).send();
       }
 
+      final String executionDuration = () {
+        final Duration? executionTime = result.executionTime;
+        if (!verbose || executionTime == null) {
+          return '';
+        }
+        final String formatted;
+        if (executionTime.inSeconds < 2) {
+          formatted = getElapsedAsMilliseconds(executionTime);
+        } else {
+          formatted = getElapsedAsSeconds(executionTime);
+        }
```

Comment:
Turbo-nit: for simple computations like this, I slightly prefer ternary expressions

```suggestion
        final String formatted = executionTime.inSeconds < 2
            ? getElapsedAsMilliseconds(executionTime)
            : getElapsedAsSeconds(executionTime);
```